app/styles/mixins/_layout.scss

```diff
@@ -0,0 +1,6 @@
+@mixin l-bottom {
```

Comment:
It's kind of nice to be able to know from looking at the CSS that something is a component. cc @noslouch